udata/tests/test_owned.py

```diff
@@ -6,15 +6,15 @@
 from udata.core.user.models import User
 
 from udata.mongo import db
-
+import udata.core.owned as owned
```

Comment:
Why not import `from udata.core.owned import Owned`?

Reply:
Because we have a class named `Owned` in this file :-)